client/public/static/js/account/my_recipes.js

```diff
@@ -339,3 +359,167 @@ $(document).ready(function () {
   const myRecipesView = new MyRecipesView();
   myRecipesView.load();
 });
+
+async function updateRecipe(recipeName) {
+  const username = utils.getUserNameFromCookie();
+  if (!username) {
+    console.error(UNABLE_TO_UPDATE_USER_NOT_LOGGED_IN);
+    utils.showAjaxAlert("Error", UNABLE_TO_UPDATE_USER_NOT_LOGGED_IN);
+    return;
+  }
+
+  const userId = await utils.getUserIdFromUsername(username);
+  if (!userId) {
+    console.error(UNABLE_TO_UPDATE_USER_NOT_LOGGED_IN);
+    utils.showAjaxAlert("Error", UNABLE_TO_UPDATE_USER_NOT_LOGGED_IN);
+    return;
+  }
+
+  window.location = "/account/update_recipe?userRecipeName=" + recipeName;
+}
+
+async function deleteRecipe(recipeName) {
+  const username = utils.getUserNameFromCookie();
```

Comment:
The checking of user name and user id is duplicated a bunch here's should create a method to handle